### ISSUE TITLE
Crashing on Windows PHP v7.3

### ISSUE THREAD

**Issue by jLynx** [krakjoe/parallel#113]
Hi

I am running PHP 64-bit 7.3.11
`PHP 7.3.11 (cli) (built: Oct 22 2019 11:20:18) ( ZTS MSVC15 (Visual C++ 2017) x64 ) Copyright (c) 1997-2018 The PHP Group Zend Engine v3.3.11, Copyright (c) 1998-2018 Zend Technologies`

Everytime I try to run the sample code PHP crashes for me. The code I am running is:
```
        $runtime = new \parallel\Runtime();

        $future = $runtime->run(function(){
            for ($i = 0; $i < 500; $i++)
                echo "*";

            return "easy";
        });

        for ($i = 0; $i < 500; $i++) {
            echo ".";
        }
        printf("\nUsing \\parallel\\Runtime is %s\n", $future->value());
```

I downloaded: `php_parallel-1.1.3-7.3-ts-vc15-x64.zip`
I have installed `php_parallel.dll` in the `php/etc `folder and `pthreadVC2.dll` in the `php `root folder

Do you know why this is and how I can fix it?

**Comment by WyriHaximus** [comment]
Do you have any error messages or traces to share, or is PHP segfaulting?

**Comment by jLynx** [comment]
Dont have any error logs that I can find. All it does it pop up with the windows message saying "CLI has crashed" or something

**Comment by krakjoe** [comment]
Heads up ... this is likely specific to windows, I'm not a windows user but do have a vm ... 

I haven't had time to look at this yet.

**Comment by ghost** [comment]
Sorry but I can't reproduce this issue with PHP 7.3.13 (or 7.2.12) on Windows 10 Pro 1909 x64. The example works fine for me.

**Comment by jLynx** [comment]
I have found an alternative way to do what I was doing. So because of that I no longer need Parellel. I am happy for this to be closed if you would like. But I was not able to resolve the issue

**Comment by krakjoe** [comment]
Looks like #109 resolved by ea3f528cf0aa283625dd4882372a89c3f2cad931
